.github/workflows/build-jenkins-plugin-release.yml

```diff
@@ -0,0 +1,19 @@
+name: Build Jenkins Plugin
```

Comment:
Shouldn't we create a separate PR for the changes in the workflow?
This allows us to also create a PR of the same branch to the original repo

Reply:
But without adding a workflow to this branch we have no way of getting the build results of this branch on Github actions.
Should we rely on the results produced by PR to jenkinsci/bitbucket-branch-source-plugin repository.

Reply:
But if we create a separate PR for this, we can merge that one first, right? Then this PR will automatically get build according the new workflow files (after optional rebasing)

Reply:
@abloemert, As there is a separate PR for workflows now, shall I remove the workflow related files from this PR and rebase this branch once the workflow related PR is merged?